Blockzilla/Info.plist

```diff
@@ -1,93 +1,98 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-    <dict>
```

Comment:
Why are there so many changes in this file? Are all of them necessary? It seems like Xcode made a lot of these changes, but we should likely only commit the ones we absolutely need.